lib/pxutil.js

```diff
@@ -205,6 +206,24 @@ function parseCookieHeader(headerValue) {
     return res;
 }
 
+function accessNestedObjectValueByStringPath(path, obj, separator='.') {
+    var properties = Array.isArray(path) ? path : path.split(separator)
+    return properties.reduce((prev, curr) => prev && prev[curr], obj);
+}
+
+function _hashString(string, hashType) {
+    try {
+        return crypto.createHash(hashType).update(string).digest("hex");
+    } catch (e) {
+        console.error("Unable to hash string with hashType", hashType);
+        return "";
+    }
+}
+
+function sha256(string) {
```

Comment:
do we really need this function?

Reply:
I regret, it's ok

Reply:
A fair question. I just didn't want to start pumping in random strings into the "hashType" parameter and give opportunity for future exceptions. This way, the hashString function is encapsulated and as we add more we can use dedicated functions. It may be overly cautious though.